### DISCUSSION TITLE
waline  怎么隐藏markdown 跳转github图标 || waline How to hide markdown and jump to github icon

### DISCUSSION THREAD

**Discussion by Jehol1983** [walinejs/waline#2093] [Q&A] [answered]
### Describe the bug | 问题描述

<img width="48" alt="image" src="https://github.com/walinejs/waline/assets/75714146/8a3c6391-f0ca-4aa6-9dab-a7d324e359d6">

### Website URL | 问题网站

wu

### Where your waline deploy? | 服务部署在哪里？

Vercel (Default)

### Where your comment data store? | 数据存储在哪里？

LeanCloud(https://leancloud.app)

---
### Describe the bug | Problem description

<img width="48" alt="image" src="https://github.com/walinejs/waline/assets/75714146/8a3c6391-f0ca-4aa6-9dab-a7d324e359d6">

### Website URL | Problem website

wu

### Where your waline deploy? | Where is the service deployed?

Vercel (Default)

### Where your comment data store? | Where is the data stored?

LeanCloud(https://leancloud.app)

**Comment by lizheming** [comment, marked as answer]
你可以自行通过 CSS 隐藏

---
You can hide
